### PR TITLE
feat: render static html from code bus

### PR DESCRIPTION
**Pull request by tripodsan**

### CODE REVIEW

File: test/rendering.test.js [hunk 1]
```diff
@@ -458,7 +458,7 @@ describe('Rendering', () => {
         'access-control-allow-origin': '*',
         'content-type': 'text/html; charset=utf-8',
         'last-modified': 'Fri, 30 Apr 2021 03:47:18 GMT',
-        'x-surrogate-key': 'zxdhoulVcSRWb0Ky ZHQXDa0L7jSHQHPX foo-id_metadata super-test--helix-pages--adobe_head',
+        'x-surrogate-key': 'zxdhoulVcSRWb0Ky SUhNxkR2spoxY489 foo-id_metadata super-test--helix-pages--adobe_head',
```
Comment (tripodsan) [inline test/rendering.test.js:461]: there was an error in the path computation for foldermapped paths... now it takes the path of the new target, which is correct

File: test/utils/path.test.js [hunk 2]
```diff
@@ -71,8 +71,8 @@ describe('Path Utils Test - getPathInfo', () => {
     assert.deepStrictEqual(getPathInfo('/express.html'), {
       selector: '',
       extension: '.html',
-      path: '/express',
-      resourcePath: '/express.md',
+      path: '/express.html',
+      resourcePath: '/express.html',
```
Comment (tripodsan) [inline test/utils/path.test.js:75]: .html requests are now handled differently than extensionless

File: test/utils/path.test.js [hunk 3]
```diff
@@ -81,8 +81,8 @@ describe('Path Utils Test - getPathInfo', () => {
 
     assert.deepStrictEqual(getPathInfo('/express.md'), {
       selector: '',
-      extension: '.html',
-      path: '/express',
+      extension: '.md',
+      path: '/express.md',
```
Comment (tripodsan) [inline test/utils/path.test.js:85]: and so are `.md`

File: test/utils/path.test.js [hunk 4]
```diff
@@ -156,6 +156,17 @@ describe('Path Utils Test - getPathInfo', () => {
       unmappedPath: '',
     });
 
+    assert.deepStrictEqual(getPathInfo('/en/express/index.plain.html'), {
+      selector: 'plain',
+      extension: '.html',
+      path: '/en/express/index.plain.html',
+      resourcePath: '/en/express/index.md',
```
Comment (tripodsan) [inline test/utils/path.test.js:163]: but `.plain` are still treated like extnesionless

**Review by stefan-guggisberg — APPROVED** [review]

**Pull request merged**